localstack-core/localstack/services/sns/v2/models.py

```diff
@@ -192,5 +193,7 @@ class SnsStore(BaseStore):
 
     TAGS: TaggingService = CrossRegionAttribute(default=TaggingService)
 
+    PHONE_NUMBERS_OPTED_OUT: list[PhoneNumber] = CrossRegionAttribute(default=list)
```

Comment:
question: is this in uppercase because it is supposed to be a constant? if that's the case, does it make sense to be in the store? 

if it's not a constant, then maybe it could be a lowercase variable